package.json

```diff
@@ -136,6 +136,7 @@
     "rollup-plugin-replace": "2.2.0",
     "rollup-plugin-uglify": "6.0.4",
     "scriptjs": "2.5.9",
+    "search-insights": "https://pkg.csb.dev/algolia/search-insights.js/commit/761a3f68/search-insights",
```

Comment:
lgtm pending this :)

Reply:
updated https://github.com/algolia/instantsearch.js/pull/4712/commits/2ad3cc07fda94ebe7a5662a3097228ddae9e6758